OpenEdXMobile/src/main/java/org/edx/mobile/viewModel/CourseDateViewModel.kt

```diff
@@ -26,8 +28,8 @@ class CourseDateViewModel @Inject constructor(
     private val repository: CourseDatesRepository
 ) : ViewModel() {
 
-    private val _syncLoader = MutableLiveData<Boolean>()
-    val syncLoader: LiveData<Boolean>
+    private val _syncLoader = MutableLiveData<Event<Boolean>>()
```

Comment:
in some cases, The app shows multiple dialogs to update the calendar event, to fix that issue, Add `Event` to the `courseDates` MutableLiveData.